tests/test_wcsUtils.py

```diff
@@ -99,25 +99,27 @@ def testNoDistortion(self):
         focalPlaneToFieldAngle = self.makeAffineTransform(scale=self.radPerMm)
         wcs = makeDistortedTanWcs(
             tanWcs=self.tanWcs,
-            pixelToFocalPlane=self.focalPlaneToPixel.getInverse(),
+            pixelToFocalPlane=self.pixelToFocalPlane,
             focalPlaneToFieldAngle=focalPlaneToFieldAngle,
         )
         self.assertWcsAlmostEqualOverBBox(wcs, self.tanWcs, bbox=self.bbox)
 
     def testDistortion(self):
         """Test makeDistortedTanWcs using a non-affine transform for pixelToFocalPlane
         """
-        # Compute a distorted wcs that matches self.tanWcs at the center
-        # of the field
-        focalPlaneToFieldAngle = afwGeom.makeRadialTransform([0.0, self.radPerMm, 0.0, self.radPerMm])
+        # Compute a distorted wcs that matches self.tanWcs at the center of the field;
+        # the amount of distortion is 10s of pixels over the detector
+        fieldAngleToFocalPlane = afwGeom.makeRadialTransform([0.0, 1/self.radPerMm, 0.0, 1000/self.radPerMm])
```

Comment:
Ah the joys of picking a "reasonable" fake model. This seems like a better choice.

Reply:
It gives much more reasonable values, and is very similar to what LSST uses (the factor of 1000 is not in the LSST model, but the LSST model only shows significant distortion at much larger focal plane radius; I tuned it to get the distortion I wanted over the range of values this test uses).